tests/unit/test_utils.py

```diff
@@ -409,3 +410,25 @@ def test_can_escape_recursive_shapes(self):
         }
         self.walker.walk(self.get_shape_model('Recursive'), self.visitor)
         self.assert_visited_shapes(['Recursive'])
+
+
+@pytest.fixture(scope='module')
+def argument_model():
+    return botocore.model.Shape('', {'type': ''})
+
```

Comment:
It might be worth running `flake8` over the PR. Noticed that there is only one new line between this method and class and there should be two.